package.json

```diff
@@ -30,10 +30,11 @@
     "ember-get-config": "^0.2.2",
     "ember-in-viewport": "2.1.1",
     "ember-scrollable": "^0.4.7",
-    "ember-truth-helpers": "1.3.0",
+    "ember-truth-helpers": "^2.0.0",
     "ember-wormhole": "^0.5.1"
   },
   "devDependencies": {
+    "@html-next/vertical-collection": "^1.0.0-beta.8",
```

Comment:
Is this intentionally only a `devDependency`, because not every user will want to try occlusion rendering just yet?

Reply:
I think this was a blunder on my part... will update when I update to `beta.9`